Packs/mnemonicMDR/Integrations/ArgusManagedDefence/ArgusManagedDefence.py

```diff
@@ -501,6 +500,10 @@ def append_demisto_entry_to_argus_case(case_id: int, entry: Dict[str, Any]) -> N
         file_name, file_extension = os.path.splitext(full_file_name)
         file_name = f"{file_name}{ATTACHMENT_SUBSTRING}{file_extension}"
         mime_type = mimetypes.guess_type(full_file_name)
+        if not mime_type[0]:
```

Comment:
Is this access safe? Or `mime_type` can be `None`?

Reply:
If mimetypes.guess_type() is not able to determine format, it will return a tuple None, None. I am going to improve this later. I found a bug on friday.

Reply:
OK, please inform me when it's ready.

Reply:
Could you please approve this merge, and the new functionality will come in a new PR.